packages/core/test/lib/commands/config.js

```diff
@@ -11,12 +11,9 @@ describe("config", function () {
   let output = "";
   let memStream;
 
-  before(function () {
+  beforeEach(function () {
```

Comment:
Clearing the cache allowed this to pass before?

Reply:
I think so - I think the complications arose from the order in which `Config` merges in changes in `load`